### ISSUE TITLE
Android Gradle plugin breaks AndroidStudio?

### ISSUE THREAD

**Issue by dennisguse** [OpenTracksApp/OpenTracks#1106]
.... reported by upgrading Android Gradle plugin to 7.1.

**Comment by dennisguse** [comment]
... and the OpenTracks cannot be build anymore, because the Android Gradle plugin now expects to have an app directory.
https://developer.android.com/studio/releases/gradle-plugin

Just one of those moments...

@pstorch do you any suggestions beside moving the code into an `app`-folder?
Or reverting the upgrade?

**Comment by dennisguse** [comment]
I am confused: executing gradle from commandline (also in CI) works.
Executing it via AndroidStudio 2022.1.1 leads to `Project 'OpenTracks' not found in root project 'OpenTracks'.`.

Downgrading the plugin and AndroidStudio works again.

**Comment by rgmf** [comment]
@dennisguse I don't know if it's the same thing, but I've also run on troubles with AndroidStudio and Gradle (it couldn't build OpenTracks) and what I did was download AndroidStudio again (2022.1.1) and install it again.

After that, It works.

**Comment by dennisguse** [comment]
"Have you tried turning it off and on again" :D
Will give it a try and hope for the best

**Comment by pstorch** [comment]
It's working for me (now).
First I had to upgrade AndroidStudio. Struggled a bit because of "no space left on device". Yeah, did a lot with VMs and Containers lately :wink: 
After that was solved and I pulled the latest commits for OpenTracks, AS told me to upgrade gradle wrapper to 7.2 (which already was) and reimport the project. I clicked the link to perform the action and it is building and running the app without any issues.
